src/qtt/measurements/scans.py

```diff
@@ -1948,6 +1946,21 @@ def acquire_segments(station, parameters, average=True, mV_range=2000,
 
     return alldata
 
+def _is_m4i(instrument):
+    try:
+        is_m4i = isinstance(instrument, qcodes.instrument_drivers.Spectrum.M4i.M4i)
+        if is_m4i:
+               warnings.warn('please use qcodes_contrib_drivers.drivers.Spectrum.M4i instead of qcodes.instrument_drivers.Spectrum.M4i.M4i)')
```

Comment:
line too long
```suggestion
               warnings.warn(f'please use qcodes_contrib_drivers.drivers.Spectrum.M4i'
                             f' instead of qcodes.instrument_drivers.Spectrum.M4i.M4i')
```